frontend/src/components/opportunity/OpportunityDocuments.tsx

```diff
@@ -0,0 +1,76 @@
+import { useTranslations } from "next-intl";
+import { Link, Table } from "@trussworks/react-uswds";
+
+interface OpportunityDocument {
+  opportunity_attachment_type: string;
+  file_name: string;
+  download_path: string;
+  updated_at: string;
+}
+
+interface OpportunityDocumentsProps {
+  documents: OpportunityDocument[];
+}
+
+const OpportunityDocuments = ({ documents }: OpportunityDocumentsProps) => {
+  const t = useTranslations("OpportunityListing.documents");
+
+  const documentTable = (documents_filtered: OpportunityDocument[]) => {
+    return (
+      <>
+        <thead>
+          <tr>
+            <th scope="col">{t("table_col_category")}</th>
+            <th scope="col">{t("table_col_file_name")}</th>
+            <th scope="col">{t("table_col_last_updated")}</th>
+          </tr>
+        </thead>
+        <tbody>
+          {documents_filtered.map((document, index) => (
+            <tr key={index}>
+              <th data-label={t("table_col_category")} scope="row">
+                {document.opportunity_attachment_type ===
+                "notice_of_funding_opportunity"
+                  ? t("type.funding_details")
+                  : t("type.other")}
+              </th>
+              <td data-label={t("table_col_file_name")}>
+                <Link target="_blank" href={document.download_path}>
+                  {document.file_name}
+                </Link>
+              </td>
+              <td data-label={t("table_col_last_updated")}>
+                {new Date(document.updated_at).toDateString() +
+                  " " +
+                  new Date(document.updated_at).toTimeString()}
+              </td>
+            </tr>
+          ))}
+        </tbody>
+      </>
+    );
+  };
+
+  const related_docs = documents.filter(
+    (d) => d.opportunity_attachment_type === "notice_of_funding_opportunity",
```

Comment:
maybe a bigger discussion for the team but I'm very much of the opinion that JS variable names should be descriptive. I'd advocate for `document` rather than `d` here.

Reply:
d was meant to be short for document, but if we want, I can change it?

Reply:
again, just one guy's opinion, and a larger conversation to be had, I like Uncle Bob's take on variable naming. 

https://dzone.com/articles/naming-conventions-from-uncle-bobs-clean-code-phil

In this case, though, we can avoid the discussion entirely, and just destructure the one attribute you need, which I think is the best way in any case:

```suggestion
    ({ opportunity_attachment_type }) => opportunity_attachment_type === "notice_of_funding_opportunity",
```